api/mod/src/database/database.py

```diff
@@ -0,0 +1,23 @@
+"""
+This module defines database config and access methods
+"""
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from mod.src.settings import SETTINGS
+
+Base = declarative_base()
+
+engine = create_engine(SETTINGS.database_url, connect_args={"check_same_thread": False})
```

Comment:
There is a comment in the fastapi docs, but in short: It disables a check in sqlite, that prevents different threads from using the same database connection. But fastapi might use multiple threads to handle the same request, so this sqlite feature would break that (I guess). (https://fastapi.tiangolo.com/tutorial/sql-databases/#note)